compatibility/sandbox-migration/util.bzl

```diff
@@ -36,3 +39,18 @@ def migration_test(name, versions, tags, quick_tags, **kwargs):
         args = ["--append-only"] + versions,
         **kwargs
     )
+
+    # Oracle was introduced after the switch to the append-only schema
+    native.sh_test(
+        name = "{}-oracle".format(name),
+        srcs = ["//sandbox-migration:test.sh"],
+        deps = ["@bazel_tools//tools/bash/runfiles"],
+        tags = tags,
+        data = [
+            "//sandbox-migration:sandbox-migration-runner",
+            "//sandbox-migration:migration-model.dar",
+            "//sandbox-migration:migration-step",
+        ] + [dep for ver in oracle_versions(versions) for dep in runfiles(ver)],
```

Comment:
maybe worth adding a check here that `oracle_versions(versions)` contains 2 elements and if not, don’t include this test.